app/src/main/java/net/squanchy/tweets/TweetsPageView.java

```diff
@@ -69,32 +73,40 @@ protected void onAttachedToWindow() {
         initList();
     }
 
+    @Override
+    protected void onDetachedFromWindow() {
+        super.onDetachedFromWindow();
+        subscription.dispose();
+    }
+
     private void initList() {
```

Comment:
This should be before `onDetachedFromWindow`, but... can we move:
* the injection out of here and into the ctor
* the adapter creation in `onFinishInflate()`
* and inline everything else in `onAttachedToWindow()`?